app/views/your_contact_details/your_contact_details.scala.html

```diff
@@ -85,6 +85,9 @@
     ))
 
     <p class="govuk-body govuk-!-padding-bottom-9" id="contact-report-change-link">
```

Comment:
You may need to move this style class to newTabLink once newTabLink is updated and message key 'cf.contact-details.report-change-pre-link-text' is updated